scripts/scenarios/cloud8/cloud8-5nodes-compute-ha.yml

```diff
@@ -36,9 +35,6 @@ proposals:
       - @@controller1@@
       - @@controller2@@
       - @@controller3@@
-      pacemaker-remote:
```

Comment:
I have no idea how the "remotes" thing works but are you sure it's OK to remove it from here (and below)?

Reply:
IIUC, this is how the compute HA feature works and it's not a feature that we configure in our "default" HA Crowbar setup.
I reverted the changes to these files (which aren't used anywhere as far as I can tell) and created two new ones, to be used explicitly for the new CI.